pytype/tools/analyze_project/main.py

```diff
@@ -59,10 +61,10 @@ def main():
   # Command line arguments overwrite file options.
   conf.populate_from(args)
   if not conf.pythonpath:
-    conf.pythonpath = environment.compute_pythonpath(args.filenames)
+    conf.pythonpath = environment.compute_pythonpath(args.inputs)
   logging.info('\n  '.join(['Configuration:'] + str(conf).split('\n')))
 
-  if not args.filenames:
+  if not args.inputs:
     parser.parser.print_usage()
```

Comment:
while you're at it, add a print statement here saying "No input files" - just printing out the usage can be unhelpful, especially if we add inputs to the config file.

Reply:
Done.